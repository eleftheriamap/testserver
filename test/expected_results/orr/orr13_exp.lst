
test/exp/test/gen/orr/orr13_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	110370e7 	add	w7, w7, #0xdc
   4:	915616f7 	add	x23, x23, #0x585, lsl #12
   8:	aa577ceb 	orr	x11, x7, x23, lsr #31
   c:	8a000000 	and	x0, x0, x0
