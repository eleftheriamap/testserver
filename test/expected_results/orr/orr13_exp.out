Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 00000000000000dc
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 00000000000000dc
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000585000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x110370e7
0x00000004 : 0x915616f7
0x00000008 : 0xaa577ceb
0x0000000c : 0x8a000000
